posthog/tasks/exports/csv_exporter.py

```diff
@@ -115,10 +115,7 @@ def _convert_response_to_csv_data(data: Any) -> List[Any]:
             # RETENTION PERSONS LIKE
             csv_rows = []
             for item in items:
-                line = {
-                    "person": item["person"].get("properties").get("email")
-                    or item["person"].get("properties").get("id")
-                }
+                line = {"person": item["person"]["name"]}
```

Comment:
This will break for users who do rely on `email` or `id`, right? We just need to use `team.person_display_name_properties` – that's configurable in project settings, so always what the user needs (and aligned between backend and frontend)

Reply:
The name property is built with this method:

```python
def get_person_name(person: Person) -> str:
    if person.properties.get("email"):
        return person.properties["email"]
    if len(person.distinct_ids) > 0:
        # Prefer non-UUID distinct IDs (presumably from user identification) over UUIDs
        return sorted(person.distinct_ids, key=is_anonymous_id)[0]
    return person.pk
```
I'm adding the requested changes in a staked PR: https://github.com/PostHog/posthog/pull/15198.